decidim-core/lib/decidim/core/engine.rb

```diff
@@ -55,6 +55,159 @@ class Engine < ::Rails::Engine
+
+        Decidim.icons.register(name: "user-line", icon: "user-line", resource: "core", category: "system", description: "", engine: :core)
+        Decidim.icons.register(name: "mic-line", icon: "mic-line", resource: "core", category: "system", description: "", engine: :core)
+        Decidim.icons.register(name: "coin-line", icon: "coin-line", resource: "core", category: "system", description: "", engine: :core)
+        Decidim.icons.register(name: "external-link-line", icon: "external-link-line", resource: "core", category: "system", description: "", engine: :core)
+        Decidim.icons.register(name: "arrow-down-s-fill", icon: "arrow-down-s-fill", resource: "core", category: "system", description: "", engine: :core)
+        Decidim.icons.register(name: "arrow-up-s-fill", icon: "arrow-up-s-fill", resource: "core", category: "system", description: "", engine: :core)
+        Decidim.icons.register(name: "treasure-map-line", icon: "treasure-map-line", resource: "core", category: "system", description: "", engine: :core)
+        Decidim.icons.register(name: "chat-new-line", icon: "chat-new-line", resource: "core", category: "system", description: "", engine: :core)
+        Decidim.icons.register(name: "draft-line", icon: "draft-line", resource: "core", category: "system", description: "", engine: :core)
+
+        # cardCell + Conference
+        Decidim.icons.register(name: "user-voice-line", icon: "user-voice-line", resource: "core", category: "system", description: "", engine: :core)
+
+        # Attachments
+        Decidim.icons.register(name: "file-text-line", icon: "file-text-line", resource: "core", category: "system", description: "", engine: :core)
+        Decidim.icons.register(name: "scales-2-line", icon: "scales-2-line", resource: "core", category: "system", description: "", engine: :core)
+        Decidim.icons.register(name: "image-line", icon: "image-line", resource: "core", category: "system", description: "", engine: :core)
+        Decidim.icons.register(name: "error-warning-line", icon: "error-warning-line", resource: "core", category: "system", description: "", engine: :core)
+        Decidim.icons.register(name: "folder-open-line", icon: "folder-open-line", resource: "core", category: "system", description: "", engine: :core)
+
+        Decidim.icons.register(name: "calendar-todo-line", icon: "calendar-todo-line", resource: "participatory_space", category: "system", description: "", engine: :core)
+        Decidim.icons.register(name: "home-8-line", icon: "home-8-line", resource: "core", category: "system", description: "", engine: :core)
+
+        Decidim.icons.register(name: "like", icon: "heart-add-line", description: "Like", category: "action", resource: "core", engine: :core)
+        Decidim.icons.register(name: "dislike", icon: "dislike-line", description: "Dislike", category: "action", resource: "core", engine: :core)
+
+        # Refactor
```

Comment:
What means this comments? Like "To be refactored" ?

Reply:
I have changed the text there ... There are some calls that are duplicated, and i would make sure they are unique...